services/auth/src/aws.js

```diff
@@ -86,9 +89,12 @@ export const awsBuilder = builder => builder.declare({
       { bucket });
   }
 
-  const sts = new AWS.STS({
-    accessKeyId: bucketCreds.accessKeyId,
-    secretAccessKey: bucketCreds.secretAccessKey,
+  const sts = new STSClient({
+    credentials: {
+      accessKeyId: bucketCreds.accessKeyId,
+      secretAccessKey: bucketCreds.secretAccessKey,
+    },
+    region: 'us-east-1',
```

Comment:
was this region a default before?

Reply:
It's hard to say. The reason it's there now is because the SDK would throw a `Region is missing` error back to me if I didn't include it.